plugins/action_generate_review_assignment_records/src/databaseMethods.ts

```diff
@@ -48,18 +47,17 @@ const databaseMethods = (DBConnect: any) => ({
       // but assignment status remains unchanged.
       const text = `
         INSERT INTO review_assignment (
-          reviewer_id, stage_id,
-          stage_number, status, application_id,
-          template_section_restrictions, level_number, is_last_level,
-          organisation_id
+          reviewer_id, stage_id, stage_number,
+          status, organisation_id, application_id,
+          allowed_sections, level_number, is_last_level
           )
         VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)
         ON CONFLICT (reviewer_id,${
           orgId ? ' organisation_id,' : ''
         } stage_number, application_id, level_number)
           WHERE organisation_id IS ${orgId ? 'NOT ' : ''}NULL
         DO
-          UPDATE SET template_section_restrictions = $6
+          UPDATE SET allowed_sections = $7
```

Comment:
I think `organisation_id` has to be last, since it's optional, so the replacement numbers get out of whack if it's in the middle. (I think)

Reply:
Ahhh, I see. Sorry I messed this up. Will fix now